extensions/sd_api_pictures/script.py

```diff
@@ -14,13 +14,15 @@
 
 # parameters which can be customized in settings.json of webui  
```

Comment:
Can this comment be expanded with example, how to add these to settings.json?

Reply:
Yup, in the planned Readme